dist/index.js

```diff
@@ -10246,7 +10246,12 @@ function getVersionFromSpec(spec, versions, maybePrependWithV0) {
   if (maybePrependWithV0 && v != null) {
     v = maybePrependWithV(v)
   }
-  return v
+
+  if (versions.includes(v)) {
+    return v
+  }
+
+  return null
```

Comment:
```suggestion

  if (!versions.includes(v)) {
    v = null
  }

  return v
```

Reply:
I'd prefer to have a single `return` here, to ease future analysis. You'll need to run `npm run build-dist` again.